doc/source/tutorials/animations.rst

```diff
@@ -0,0 +1,150 @@
+.. _animations:
+
+.. index:: animations
+
+How to create an animation using Emperor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this tutorial we describe how to create a principal coordinates analysis
+(PCoA) plot, and display animated traces of the samples sorted by a metadata
+category.
+
+To do this, we need to have two metadata categories, a *gradient* category, and
+a *trajectory* category. The *gradient* category determines the order in which
+samples are connected together, the *trajectory* category determines how
+samples are grouped together.
+
+In most cases these metadata columns already exist as part of your sample
+information, however you may need to do some curation to make these compatible
+with Emperor. As an example, in a longitudinal study where you wish to track
+the oral microbiome changes in a cohort of 3 mice over the course of 5 weeks,
+each sample will be described by the following columns:
+
+* ``cage_number``: the cage where each mice was housed, more than one mice could
+  have resided in the same cage.
+
+* ``age_in_years``: the age of each mice in years.
+
+* ``week``: the number of the week in this experiment.
+
+* ``sex``: the sex of each mice.
+
+* ``mice_identifier``: where each mice is assigned a unique identifier.
+
+In the example above, we can use the ``week`` column as our *gradient* category,
+so as long as all the values are numerical. To be more precise, a column where
+values were indicated as ``pre-treatment, first, second, third and last`` would
+not be appropriate and instead would need to be converted into (for example):
+``-1, 1, 2, 3 and 4`` (remember we have 5 weeks of data).
+
+As for the *trajectory* category, the natural choice would be to use the
+``mice_identifier`` column, because it uniquely identifies every mice, and
+should be the same throughout the experiment.
+
+.. figure:: trajectories.png
+   :alt: Cartoon representation of the example above.
+
+   Cartoon representation of the example we see above. On the left, the
+   unmodified ordination coloring samples by mice. On the center, the same
+   odination with a label for each sample, corresponding to the week where this
+   sample was collected. On the right, samples connected by a line, where
+   the order is determined by the collection time.
+
+
+In the next section we will go through an example using published data from
+`Weingarden et al. 2015 <https://www.ncbi.nlm.nih.gov/pubmed/25825673>`_.
+
+Data
```

Comment:
We don't really need both, but I think that if you are unfamiliar with the topic, it may be of more help to look at a small synthetic example with just a handful of data points. As per your other comments, I've made the distinction between these two more explicit, hopefully this will help.